pkg/client/client_venafi_cloud.go

```diff
@@ -327,7 +327,7 @@ func (c *VenafiCloudClient) sendHTTPRequest(request *http.Request, responseObjec
 
 	if response.StatusCode != http.StatusOK && response.StatusCode != http.StatusCreated {
 		body, _ := io.ReadAll(response.Body)
-		return fmt.Errorf("failed to execute http request to VaaS. Request %s, status code: %d, body: %s", request.URL, response.StatusCode, body)
+		return fmt.Errorf("failed to execute http request to VaaS. Request %s, status code: %d, body: [%s]", request.URL, response.StatusCode, body)
```

Comment:
```suggestion
		return fmt.Errorf("failed to execute http request to Venafi Control Plane. Request %s, status code: %d, body: [%s]", request.URL, response.StatusCode, body)
```

Reply:
I think nowadays we're trying to use more the VCP / Venafi Control Plane naming instead of VaaS